.github/workflows/pipeline.yml

```diff
@@ -119,18 +120,30 @@ jobs:
       working-directory: ${{env.odbodsPath}}
 
     # Commit changes
-    - name: Git config
+    - name: Git config JKAN
       working-directory: ${{env.jkanPath}}
       run: |
-        git config user.name "GitHub Actions"
-        git config user.email "<>"
-    - name: Git status
+        git config user.name 'GitHub Actions'

```

Comment:
I think if you add the `--global` flag to these commands then you only need to run them once. Documentation: https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration

Also, I was looking at the documentation recently and noticed that if you set these values then it will correctly attribute the git commit to the GitHub Actions bot:
Username: github-actions[bot]
Password: github-actions[bot]@users.noreply.github.com

![image](https://user-images.githubusercontent.com/46202639/211174577-c145a8fc-e99e-4ef2-affd-4391d2b5a9ee.png)

Reply:
added `--global` and removed redundant config. changed username and email values.